qrenderdoc/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+set(CMAKE_AUTOMOC ON)
```

Comment:
Adding a CMakeLists for this project - does this mean it will need to be maintained separately from the qrenderdoc.pro file? I'm not so happy with this, since eventually there will be a qrenderdoc project for the visual studio solution so this will mean 3 different build systems to update - I am OK with maintaining 2 separate (Makefile/VS or CMake/VS both are OK) but that is the most I will go to :smile:.

Is it possible to construct the CMakeLists to just passthrough to qmake & make instead? Since qrenderdoc doesn't need to be ported to android anyway, I guess there is less need to have a 'native' cmake build for it.